src/js/flows/ingestFiles.js

```diff
@@ -115,7 +115,7 @@ const postFiles = (client, jsonTypesPath) => ({
       let types = isEmpty(jsonTypesPath)
         ? "default"
         : await lib.file(jsonTypesPath).read()
-      stream = client.logs.post({space: spaceId, paths, types})
```

Comment:
I tried to upload some zeek logs and found that his api call hadn't been fully converted.